android/app/src/main/java/app/organicmaps/sdk/sound/TtsPlayer.java

```diff
@@ -351,8 +349,18 @@ private boolean getUsableLanguages(List<LanguageData> outList)
     return res;
   }
 
+  @NonNull
+  private List<Pair<String, String>> getSupportedLanguages() {
+    if (sSupportedLanguages == null) {
+      sSupportedLanguages = nativeGetSupportedLanguages();
+    }
```

Comment:
This block doesn't look clang-formatted.
```suggestion
  private List<Pair<String, String>> getSupportedLanguages()
  {
    if (sSupportedLanguages == null)
      sSupportedLanguages = nativeGetSupportedLanguages();
```

Reply:
My bad. You should consider enabling (and probably fixing) clang-format check